user/contactmethod/store.go

```diff
@@ -194,9 +192,9 @@ func (s *Store) DeleteTx(ctx context.Context, tx *sql.Tx, ids ...string) error {
 		return err
 	}
 
-	uids := make([]uuid.UUID, len(ids))
-	for i, id := range ids {
-		uids[i] = uuid.MustParse(id)
+	uids, err := validate.ParseManyUUID("UserIDs", ids, len(ids))
```

Comment:
Sorry, I meant you should replace line 181 (to avoid calling validate twice)

e.g., change line 181 from this:
```
err = validate.ManyUUID("ContactMethodID", ids, 50)
```
to this:
```
uids, err = validate.ParseManyUUID("ContactMethodID", ids, 50)
```
and then you can get rid of line 195 here

Reply:
looks like you can also get rid of lines 187-190

Reply:
oops missed that one, thanks